src/Symfony/Component/Notifier/Bridge/Clickatell/ClickatellTransportFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Clickatell;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Kevin Auvinet <k.auvinet@gmail.com>
+ */
+final class ClickatellTransportFactory extends AbstractTransportFactory
+{
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('clickatell' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'clickatell', $this->getSupportedSchemes());
+        }
+
+        $authToken = $this->getUser($dsn);
+        $from = $dsn->getRequiredOption('from');
```

Comment:
```suggestion
        $from = $dsn->getOption('from', '');
```

It looks like this is not really required, isn't it?